### PR TITLE
add german calendar icons

### PR DESCRIPTION
**Pull request by starmaerker**

### CODE REVIEW

**Comment by starmaerker** [comment]
[Find German calendar icons](https://trello.com/c/rMNkEsc3/74-find-german-calendar-icons)

**Comment by starmaerker** [comment]
[Adjust link color](https://trello.com/c/FgWAL4Gz/73-adjust-link-color)

File: static/css/style.css [hunk 1]
```diff
@@ -12,6 +12,10 @@ main {
   flex: 1;
 }
 
+a {
+  color: #2B6550;
```
Comment (jarjan) [inline static/css/style.css]: Better change all color related stuff through modifying theme.

Comment (starmaerker) [reply static/css/style.css]: ok

**Pull request merged**